pkg/rpcclient/unwrap/unwrap_test.go

```diff
@@ -97,6 +101,12 @@ func TestStdErrors(t *testing.T) {
 			require.Error(t, err)
 		}
 	})
+	t.Run("HALT state with empty stack", func(t *testing.T) {
+		for _, f := range funcs {
+			_, err := f(&result.Invoke{State: "HALT"}, nil)
+			require.Error(t, err)
+		}
+	})
 	t.Run("multiple return values", func(t *testing.T) {
```

Comment:
This test passes, but for a wrong reason. I am not sure what to do with it, don't want to complicate the array of testcases, it is beautiful.

Reply:
I see you have it covered, so ok.